cheatsheets/AJAX_Security_Cheat_Sheet.md

```diff
@@ -2,7 +2,7 @@
 
 This document will provide a starting point for AJAX security and will hopefully be updated and expanded reasonably often to provide more detailed information about specific frameworks and technologies.
 
-## Client Side (Javascript)
+## Client Side (JavaScript)
```

Comment:
There are files named Javascript. I'm intentionally skipping them as that would require dealing w/ the assets directory which I intentionally skipped (to avoid damaging their contents).

The brand is `JavaScript` like `PowerPoint` (unlike `Javadoc` which abandoned its casing; and `Micro-Soft` which long since abandoned its hypen and casing).